markdown/3.x/en/ff-checkout-tracking.md

```diff
@@ -4,15 +4,17 @@ The `ff-checkout-tracking` element sends out a checkout tracking request
 for each `ff-checkout-tracking-item` in its DOM. These elements are supposed
 to be generated on server side after the checkout is completed.
 Each product needs its own `ff-checkout-tracking-item` with its respective
-`record-id` and `count`.
+`record-id`. Optional you can also provide a `count` and a `price` attribute.
 
+The `count` attribute specifies how many items of it were bought. It defaults to 1.
```

Comment:
Shouldn't it be:
"The `count` attribute specifies how many items of the product were bought."? This `it` pronoun refers to some undefined noun here and I had to read twice to understand. Or maybe I'm wrong and omitted something?

Reply:
Being a bit more explicit should help reduce ambiguity.